core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy

```diff
@@ -32,23 +32,16 @@ st = namespace("jelly:stapler")
 f.section(title: _("Administrative monitors configuration")) {
     f.advanced(title: _("Administrative monitors")) {
         f.entry(title: _("Enabled administrative monitors")) {
-            p(_("blurb"))
-            div(width: "100%") {
-                for (AdministrativeMonitor am : new ArrayList<>(AdministrativeMonitor.all())
-                        .sort({ o1, o2 -> o1.getDisplayName() <=> o2.getDisplayName() })) {
-                    f.block() {
-                        f.checkbox(name: "administrativeMonitor",
-                                title: am.displayName,
-                                checked: am.enabled,
-                                json: am.id)
-                        if (am.isSecurity()) {
-                            span(style: 'margin-left: 0.5rem', class: 'am-badge', _("Security"))
-                        }
-                    }
-                    div(class: "tr") {
-                        div(class: "setting-description") {
-                            st.include(from: am, page: "description", optional: true)
```

Comment:
Removed without replacement?

Reply:
Restored in https://github.com/jenkinsci/jenkins/pull/5923/commits/78735a6fe9c08013d8e54a8a2f1e2993ccc03927